app/views/category/index.html.erb

```diff
@@ -34,13 +34,21 @@
 
 
 <%= form_tag( '/category/form' , remote: true, method: 'post') do |f| %>
+
     <div class = "columns">
     <div class = "column">
+    
     <ul>
     <% @columns1.each do |x| %>
     <li>
-      <%= label_tag "Category #{x}" %>
-      <%= check_box_tag "category[]", "category#{x}" %>
+    <div id="check-button">
+      <label>
+         <%= check_box_tag "category[]", "category#{x}"  %>
+          <span><%= label_tag "Category #{x}" %></span>  
```

Comment:
Phew this took me a while to figure out, but you may have noticed that your buttons are a little difficult to toggle if you click directly on the buttons text. If you remove the label nested inside the `<span>` it fixes this issue.

So rather than nesting the label inside the `<span>`:
`<span><%= label_tag "Category #{x}" %></span>`

Just render the value as a plain string:
`<span> Category <%= x %> </span>`